### PR TITLE
Steam 968 UI changes

### PR DESCRIPTION
**Pull request by julianxcarter**
Makes a few changes requested by Michelle:
1. Times and time zones are included with date times that include that information.
2. Dose to Volume tables now include the "volume Label" in the title of each column. 
3. Modalities and Techniques have been split off into their own table. One modality can have several techniques, which will be separated by new lines. 
4. Tables have been reordered per Michelle's suggestions.

### CODE REVIEW

File: src/lib/mappingUtils.js [hunk 1]
```diff
@@ -8,20 +8,30 @@ function getProcedureIntent(resource, resourceType) {
...
+    `${resourceType}.extension.where(url = 'http://hl7.org/fhir/us/mcode/StructureDefinition/mcode-radiotherapy-modality-and-technique')`
+  );
+
+  return extensions.map((extension) => {
+    return {
+      Modality: extension.extension.find(
+        (extension) =>
+          extension.url ===
+          "http://hl7.org/fhir/us/mcode/StructureDefinition/mcode-radiotherapy-modality"
+      ).valueCodeableConcept.coding[0].display,
```
Comment (dmendelowitz) [inline src/lib/mappingUtils.js]: Do we need the `?` operator here like we added to a lot of other functions in this file?

File: src/lib/mappingUtils.js [hunk 2]
```diff
@@ -8,20 +8,30 @@ function getProcedureIntent(resource, resourceType) {
...
+      Techniques: extension.extension
+        .filter(
+          (extension) =>
+            extension.url ===
+            "http://hl7.org/fhir/us/mcode/StructureDefinition/mcode-radiotherapy-technique"
+        )
+        .map((techniqueObj) => {
+          return techniqueObj.valueCodeableConcept.coding[0].display;
```
Comment (dmendelowitz) [inline src/lib/mappingUtils.js:30]: The `?` question here as well

**Review by dmendelowitz — CHANGES_REQUESTED** [review]
This all looks good for the most part, just a few small comments

**Comment by julianxcarter** [comment]
@dmendelowitz Good catch! I added the optional chining operator throughout the `getModalities` function in commit [0eba9c1](https://github.com/mcode/rttd-summary-webapp/pull/24/commits/0eba9c11aaf38d66c095d9c7ad127c517d3fe827).

**Comment by dmendelowitz** [comment]
I think rebasing caused some issues. I get errors when trying to fetch patient data when the include metadata option is unchecked, but am able to fetch data when the include metadata option is checked. I think this might be related to you adding hard-coded references to "Volume Label" to the MultiEntryDataTable component, since I used that component to show metadata for one type of resource. I think it's probably best to keep the table components generic, so we can reuse them.

**Comment by julianxcarter** [comment]
@dmendelowitz  That issue was kind of two-pronged, and commit [5774367](https://github.com/mcode/rttd-summary-webapp/pull/24/commits/5774367129efa4a9828f0693f3fedfba375e7423) should address both issues.
1. If metadata was included, the `MultiEntryDataTable` component would still look for the "Volume Label" field to populate the title. This would result in "undefined" being printed to the metadata headers which was inappropriate. I edited the component to not assume that it would always be used in the Dose to volume context. 
2. If no metadata was included at all, the app would error out because the Metadata component was actually still being rendered, just with no data (this is because empty arrays in javascript are still truthy values, so I fixed that by making the check in the `TreatmentVolumeTable` a bit more explicit). That bug can actually be seen in our currently deployed site, but it's especially nasty here because the  `MultiEntryDataTable` component was trying to access a property on those `undefined` metadata objects.

**Comment by dmendelowitz** [comment]
Confusing javascript truthyness strikes again... This all looks good now!

**Review by dmendelowitz — APPROVED** [review]

**Pull request merged**